pac4j-core/src/main/java/org/pac4j/core/client/IndirectClient.java

```diff
@@ -40,7 +40,7 @@ protected void internalInit(final WebContext context) {
     }
 
     @Override
-    public final HttpAction redirect(final WebContext context) throws HttpAction {
+    public HttpAction redirect(final WebContext context) throws HttpAction {
```

Comment:
In fact I need different callback URL by tenant (tenant1.mydomain.com, tenant2.mydomain.com)
This URL is constructed in the OAuth20Service and checked when the user come back from google login page.
If I proxify Google2Client by tenant I am sure I have one OAuth20Service by tenant too with the right callback URL.